rfs_test/TEST_bios_schema.py

```diff
@@ -0,0 +1,63 @@
+# Copyright Notice:
+# Copyright 2016-2018 Distributed Management Task Force, Inc. All rights reserved.
+# License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/Redfish-Service-Conformance-Check/blob/master/LICENSE.md
+
+#####################################################################################################
+# File: TEST_bios_schema.py
```

Comment:
We should probably remove this file completely since the BIOS resource isn't exactly interoperable

Reply:
Will be removed in the next PR.